packages/adapters/wagmi/src/client.ts

```diff
@@ -682,7 +682,12 @@ export class WagmiAdapter implements ChainAdapter {
 
     if (this.wagmiConfig) {
       if (connector) {
-        if (connector.name === 'WalletConnect' && connector.getProvider && address) {
+        if (
+          status === 'connected' &&
```

Comment:
@enesozturk @svenvoskamp why weren't we checking if this was connected? 

Might be a reason for this from the multichain side?